net-nntp/sabnzbd/sabnzbd-3.6.1-r1.ebuild

```diff
@@ -47,6 +47,8 @@ DEPEND="
 RDEPEND="
 	${DEPEND}
 	>=app-arch/par2cmdline-0.4
+	app-arch/unrar
+	app-arch/unzip
 	net-misc/wget
 "
 BDEPEND="
```

Comment:
Maybe you wanted to add them inside `BDEPEND="test? ( .. )"` like the p7zip is? This'd also make the `optfeature`s below obsolete.

Reply:
Yeah that was a whoops on my end.... 

Does it make optfeatures obsolete? I can't imagine most users are building with tests. It's also not a hard runtime dependency, up to users to decide what archives they want to support.

Reply:
No no, **if** you added them to RDEPEND like this, it'd make `optfeature` obsolete. If you add them to BDEPEND like I imagine you were supposed to, `optfeature` remains functional.

Reply:
got it, it's updated like originally intended. sorry about that